src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-windows.mdx

```diff
@@ -24,70 +24,98 @@ To run our Kubernetes installation for Windows, follow the steps below to:
 Before you install [New Relic's Kubernetes integration](/docs/integrations/kubernetes-integration/get-started/introduction-kubernetes-integration), review the [compatibility and requirements](/docs/integrations/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements).
 
 <Callout variant="important">
```

Comment:
any collaout here pointing clarifying that v3 still does not support it?

Reply:
Done! https://github.com/newrelic/docs-website/pull/5742/commits/5e1543f3cf6bb452faf660d9310bf8894ff6c501